apps/main/models.py

```diff
@@ -86,6 +88,49 @@ def __str__(self):
     sale_price = models.DecimalField(max_digits=16, decimal_places=2,
                                      null=True, blank=True, default=0)
 
+    def exists(self):
```

Comment:
The name of this method conflicts with the `exists()` method of Django Querysets and may cause misunderstanding:
https://docs.djangoproject.com/en/2.0/ref/models/querysets/#django.db.models.query.QuerySet.exists
Please rename it for clearer usage, e.g. `image_exists()`.